src/lib/containers/download_list_v2/DownloadListTable.tsx

```diff
@@ -256,25 +260,61 @@ export default function DownloadListTable() {
                         />
                       </td>
                       <td>{createdOn}</td>
-                      <td>
-                        {item.fileSizeBytes &&
-                          calculateFriendlyFileSize(item.fileSizeBytes)}
-                      </td>
-                      <td>
-                        <button
-                          className={TESTING_TRASH_BTN_CLASS}
-                          onClick={() => {
-                            removeItem({
-                              fileEntityId: item.fileEntityId,
-                              versionNumber: item.versionNumber,
-                            })
-                          }}
+                      <td className="stickyColumn">
+                        <span className="downloadItem">
+                          <DirectDownload
+                            associatedObjectId={item.fileEntityId}
+                            associatedObjectType={FileHandleAssociateType.FileEntity}
+                            entityVersionNumber={item.versionNumber.toString()}
+                            displayFileName={false}
+                            onClickCallback={() => {
+                              removeItem({
+                                fileEntityId: item.fileEntityId,
+                                versionNumber: item.versionNumber,
+                              })
+                            }}
```

Comment:
Not sure if there's more to this in the design coming later, but IMO there needs to be more feedback that the item will be removed from the list if you download it individually.

Reply:
That's a great point.  For now, maybe I'll pop up a message to elicit recommendations from the Design team

Reply:
May need to let you know that this will happen before clicking on the individual download, but let's test it!